properties/base.py

```diff
@@ -203,14 +204,18 @@ def _validate_props(self):
             self._validating = False
         return True
 
-    def serialize(self, using='json'):
+    def serialize(self):
```

Comment:
Yeah, for now all the built in serialization uses JSON, and the way it's built now is more favourable to defining alternative `Property` `serializer`s and possibly overriding `HasProperties` `serialize` rather than building in fixed alternatives to JSON.

That said, I'm totally happy to put `using='json'` back in for future extensibility if you like.